src/sindri/sindri.py

```diff
@@ -123,12 +123,12 @@ def _hit_api(self, method: str, path: str, data=None, files=None) -> tuple[int,
         try:
             if method == "POST":
                 response = requests.post(
-                    full_path, headers=self.headers_json, data=data, files=files
+                    full_path, headers=self.headers_json, data=data, files=files, verify=False
```

Comment:
What's the purpose of `verify=False` here? Isn't this opening people up to MITM attacks?

Reply:
Great point. It was to make Sindri employee development a little easier if they stood up an instance of the Sindri API with fake certs. However, the security hole is not worth it.. removed in [4b6f5ef](https://github.com/Sindri-Labs/sindri-python/pull/21/commits/4b6f5ef85b5d0a339d92107ad5e560f5e70656f2)